### ISSUE TITLE
Add a stress characteristic

### ISSUE THREAD

**Issue by Lit-Any** [Lit-Any/Collabs#14]
Add a stress characteristic to the player. This causes an additional health loss every year that is offset by the comfort parameter.
Possibly stress could also play a role in relationships?

Health loss = (p.stress-p.comfort) / 10
